Tests/SwiftDocCUtilitiesTests/ConvertActionTests.swift

```diff
@@ -3251,6 +3251,7 @@ class ConvertActionTests: XCTestCase {
         
         XCTAssertEqual(fileSystem.dump(subHierarchyFrom: targetURL.path), """
         Output.doccarchive/
+        ├─ custom-scripts/
```

Comment:
Question: is it expected that we create this folder even if the user didn't pass any custom scripts?